librocksdb-sys/rocksdb_lib_sources.txt

```diff
@@ -158,6 +160,12 @@ options/options_parser.cc
 port/mmap.cc
 port/port_posix.cc
 port/stack_trace.cc
+port/win/env_default.cc
+port/win/env_win.cc
+port/win/io_win.cc
+port/win/port_win.cc
+port/win/win_logger.cc
+port/win/win_thread.cc
```

Comment:
The files related to Windows are redundant here. They are added by the `build.rs` script if a target OS is Windows.